libraries/AP_Scripting/applets/Aerobatics/FixedWing/plane_aerobatics.lua

```diff
@@ -1766,6 +2127,23 @@ function log_pose(logname, pos, quat)
                 math.deg(quat:get_euler_yaw()))
 end
 
+
+function log_position(logname, loc, quat)
+   local utc_s, utc_us = utc_micros()
```

Comment:
Its a bit odd to add a binding to get a value that it only logged. Presumably these values are logged in the log somewhere else and could be interpolated by using the log timestamp?

Reply:
i've found a way to do this without utc_micros. A bit messy, but it does work using existing GPS bindings
I still think we should add a global time function though, probably with the double precision change